packages/mdc-menu/README.md

```diff
@@ -136,6 +136,25 @@ classes:
 | `mdc-simple-menu--open-from-bottom-right` | Open the menu from the bottom right. |
 
 
+#### Disabled list items
```

Comment:
I'm not fundamentally opposed to it. Especially since there is some language around a mechanism like that in the [spec](https://material.io/guidelines/components/menus.html#menus-usage). 
 >Menu items which are relevant but need certain conditions to be met may be disabled. For example, the Copy menu option becomes enabled when text is selected.

I don't know if that's what this story is concerned with though. @traviskaufman WDYT?